HCI-lab/src/pages/index.js

```diff
@@ -1,10 +1,60 @@
 import React from "react"
 
 import Layout from "../layouts"
+import Image from "../components/image"
+import BackgroundImage from "../components/backgroundImage"
+
+import {homePageTexts, images} from "../constants"
+
+import styles from "./index.module.css"
 
 const IndexPage = () => (
   <>
     <Layout>
+      <div className={styles.titleContainer}>
+        <div className={styles.titleImage}>
+          <BackgroundImage name={images.exterior1}/>
+        </div>
+        <div className={styles.titleText}>
+          {homePageTexts.titleText}
+        </div>
+      </div>
+      <div className={styles.contentContainer}>
+        <div className={styles.contentText1}>
```

Comment:
ode malo razmislit o nekoj iteraciji po konstantama, di ce se za svaku konstantu renderirat njen markup i style, mozda ce nam bit tako lakse citat kod. To mi ovako pari na prvu, ako se ne slazes slobodno napisi misljenje 😄